util/validation/checkCnt.js

```diff
@@ -0,0 +1,8 @@
+export default function checkCnt(counts, key) {
+    if (counts.cnt > counts.maxCnt) {
```

Comment:
```suggestion
    if (cnt > maxCnt) {
        counts.moveLonger = [key];
        counts.maxCnt = cnt;
    }
     if (cnt === maxCnt) {
        moveLonger.push(key);
    }
```

요렇게 작성하는게 가독성도 좋은 것 같습니다!